generator/patacrep.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-#    Copyright (C) 2014 The Patacrep Team
+#    Copyright (C) 2015 The Patacrep Team
```

Comment:
C'est assez fourbe, car j'ai en fait créé un nouveau fichier (patacrep.py) dans lequel j'ai copié-collé une bonne partie de songs.py (fichier, que j'ai ensuite supprimé).

Du coup git pense que c'est un renommage suivi d'une modification...

---

Concernant le copyright, j'avais eu une courte formation pro, qui disait qu'il suffisait de mettre `Copyright (C) The Patacrep Team` en tête de chaque fichier (sans la licence complète) et d'avoir un fichier `LICENCE` à la racine du projet avec le texte complet de la licence.

De mémoire:
* le copyright sert juste à indiquer qui est le propriétaire (en l’occurrence du code) et devrait donc figurer dans chaque fichier
* la licence sert à indiquer les droits accordées par ce propriétaire sur son travail (mais le travail reste la possession de l'auteur (et il est possible d'accorder différentes autorisations à différentes personnes)